### PR TITLE
3D Modeling Workshop Proposal

### PR DESCRIPTION
**Pull request by thesuperRL**
Thanks for making a PR! Please fill in the below:

Name:

Do you run a club? 

What relevant experiences do you have to help you write this Jam?

Will you be available on Slack to provide support to Hack Clubbers who run into issues while going through your Jam?

### CODE REVIEW

**Comment by thesuperRL** [comment]
Name: Ryan Li

Do you run a club? Yes

What relevant experiences do you have to help you write this Jam? I have 3D modeled for 2 years, and I have even started a company selling products I model and then 3D print or Laser Cut.

Will you be available on Slack to provide support to Hack Clubbers who run into issues while going through your Jam? Yes

**Comment by Arpan-206** [comment]
Sorry, but I'm afraid we're only looking for Web based jams for the bounty.

(But, I would love to see this as a Jam since I'm looking forward to learning 3D modelling, hehe)

**Comment by SerenityUX** [comment]
This is such an awesome Jam! Is there anyway you could use software like Spline?

**Comment by SerenityUX** [comment]
I love Blender and definitely think at some point we should have a Blender Jam (it's just atm we're optimizing for Clubs and many Clubs only have access to Chromebooks)

**Comment by Arpan-206** [comment]
> I love Blender and definitely think at some point we should have a Blender Jam (it's just atm we're optimizing for Clubs and many Clubs only have access to Chromebooks)

+1 on this

File: jams/3D_Modeling_Workshop.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# 3D Modeling: Making Hardware to Complement Your Software
+
+### Details:
+_Outcome:_ Learn to 3D Model first with TinkerCAD, then Blender, and finally, AutoDesk Fusion 360
```
Comment (sahitid) [inline jams/3D_Modeling_Workshop.md]: hi! i love this idea and how you're really excited about it! however, at the moment, i feel like this is not a jam. is there any way you could incorporate this concept of 3d modeling into a project that could be done in a group/club setting?

Comment (thesuperRL) [reply jams/3D_Modeling_Workshop.md]: I can make it about maybe building a logo to be printed, or making 3D models of various objects around the school in different pieces to be joined together. 

Also, TinkerCAD has the ability to model breadboards, which can be used for electrical engineering as well.

**Comment by thesuperRL** [comment]
> Sorry, but I'm afraid we're only looking for Web based jams for the bounty.
> 
> (But, I would love to see this as a Jam since I'm looking forward to learning 3D modelling, hehe)

I just edited it, it is now web based since we use TinkerCAD instead of other programs.

File: jams/3D_Modeling_Workshop.md (hunk 1 above)
Comment (Arpan-206) [reply jams/3D_Modeling_Workshop.md]: This was one of my concerns too.

**Comment by SerenityUX** [comment]
wowowwowoowow!!! :D 

(so excited for the rest of the team to check this out)

File: jams/3D_Modeling_Workshop.md (hunk 1 above)
Comment (sahitid) [reply jams/3D_Modeling_Workshop.md]: I like those ideas! I'm glad you went back and edited your Jams to change the outcome/goal.

**Comment by Arpan-206** [comment]
Can you delete the extra file like the one that's not web based?

**Comment by thesuperRL** [comment]
> Can you delete the extra file like the one that's not web based?

I just did

File: jams/3D_Modeling_Web_Based.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+_Time estimation:_ ~1 Hour
+
+### Breakdown
+TinkerCAD
+- Set up: Login to Tinkercad
+- Part 1: Learn to make basic shapes from the sidebar, and how to model them
+- Part 2: Learn to align combine, and cut shapes
+- Part 3: Learn to import and export
+
+**Make sure your proposal (similar to the one above) answer these questions about your Jam idea:**
```
Comment (Arpan-206) [inline jams/3D_Modeling_Web_Based.md:25]: The answers to the questions below are a little short and vague, find expanding on them a little bit?

File: jams/3D_Modeling_Web_Based.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+_Language used:_ N/A
+
+_Customization opportunity:_ Make any 3D model project after learning the basics, whether you want it to look beautiful, function perfectly, or both!
+
+_Platform Limitations:_ be good enough to run a browser, and actually have a keyboard (be a computer not a mobile device) and is able to use a mouse
+
+_Time estimation:_ ~1 Hour
+
+### Breakdown
```
Comment (Arpan-206) [inline jams/3D_Modeling_Web_Based.md:17]: Can you add time estimates for each of the steps?

**Review by Arpan-206 — CHANGES_REQUESTED** [review]
Answers need to be a little more verbose.

File: jams/3D_Modeling_Web_Based.md (hunk 3 above)
Comment (thesuperRL) [reply jams/3D_Modeling_Web_Based.md:17]: I've been told I model pretty quickly, so I'm going to give pretty generous estimates

File: jams/3D_Modeling_Web_Based.md (hunk 2 above)
Comment (thesuperRL) [reply jams/3D_Modeling_Web_Based.md:25]: I don't see how they are vague. I'll expand best I can. Mind pointing me the vague portions so I can elaborate?

File: jams/3D_Modeling_Web_Based.md (hunk 3 above)
Comment (Arpan-206) [reply jams/3D_Modeling_Web_Based.md:17]: Uhh, I just wanted you to add time required for each step like
- Part 0: Learn to view items like a 3D modeler in basic shapes (10mins)

etc

**Review by Arpan-206 — CHANGES_REQUESTED** [review]

**Comment by thesuperRL** [comment]
@Arpan-206 I did, check the newest version

File: jams/3D_Modeling_Web_Based.md (hunk 3 above)
Comment (Arpan-206) [reply jams/3D_Modeling_Web_Based.md:17]: Sorry missed it

**Review by Arpan-206 — APPROVED** [review]
LGTM!

**Review by sarthaktexas — APPROVED** [review]
Lgtm

**Pull request merged**